src/components/Input/Input.vue

```diff
@@ -134,7 +136,7 @@ export default defineComponent({
       required: false
     }
   },
-  emits: ['update:modelValue', 'invalid'],
+  emits: ['update:modelValue', 'invalid', 'keyup', 'keypress', 'keydown'],
```

Comment:
I didn't think you needed to do this with vue3? Doesn't it bind any listener that doesn't match an event in the `emits` option as a native listener?

Reply:
agreed, this leads me to believe it should bind natively https://v3.vuejs.org/guide/migration/v-on-native-modifier-removed.html#_3-x-syntax

Reply:
That was my thought but linter was throwing errors - if someone wants to investigate that's fine too.

Reply:
Happy to approve for now as I don't think it's affecting anything to have them in.  But we should look into this and check in slower time.